FirebaseCombineSwift/Sources/Auth/Auth+Combine.swift

```diff
@@ -156,13 +156,13 @@
       }
     }
 
-    /// Signs in using an email address and password.
+    /// Signs in a user with the given email address and password.
```

Comment:
I agree with you. My rationale for this change and the one above was that: an Auth instance is the entity that facilitates the authentication of a user as opposed to being the entity that's getting authenticated. Im thinking this way because the only thing that is changing for the Auth instance is the currently signed in user (currentUser). So, if this is the way to think about it, then it might be more clear to phrase things like: "signs in **a user**", "creates **a user**".

cc: @morganchen12 WDTY about this and the above change?